.prettierrc

```diff
@@ -0,0 +1,10 @@
+ {
+  "useTabs": false,
+  "tabWidth": 2,
+  "arrowParens": "always",
+  "semi": true,
+  "printWidth": 100,
+  "plugins": [
+    "prettier-plugin-tailwindcss"
```

Comment:
I think we should use [Eslint plugin](https://www.npmjs.com/package/eslint-plugin-tailwindcss) instead of prettier one